lib/install.js

```diff
@@ -897,28 +902,34 @@ function clean() {
 }
 exports.clean = clean;
 
+/* Like concat, but in place */
+function into(extras) {
+  var list = this;
+  extras.forEach(function(item) { list.push(item); });
+}
+
 // depthCheck returns true to keep going (files being ignored), false to add the dir to the flat list
 function readDirWithDepth(dir, depthCheck) {
   var flatDirs = [];
-  return asp(fs.readdir)(dir)
-  .catch(function(e) {
-    if (e.code === 'ENOTDIR')
-      return;
-    throw e;
+  return asp(fs.lstat)(dir).then(function(fileInfo) {
```

Comment:
Shouldn't be an issue I don't expect, and if it turns out it is we can add a flag later. Let's not worry about this for now.